modules/core/src/v2/coins/eth.ts

```diff
@@ -593,6 +594,7 @@ export class Eth extends BaseCoin {
       const signature = Util.ethSignMsgHash(operationHash, Util.xprvToEthPrivateKey(userPrv));
 
       const txParams = {
+        isBatch: params.txPrebuild.isBatch === true,
```

Comment:
Why `=== true` instead of just the value?

Reply:
Wasn't sure if it was populated in build, made the change